app/subscribers/solidus_stripe/webhook/payment_intent_subscriber.rb

```diff
@@ -27,6 +27,32 @@ def complete_payment(event)
           )
         end
       end
+
+      # Voids a payment.
+      #
+      # Marks a Solidus payment associated to a Stripe payment intent as
+      # failed, adding a log entry about the event.
+      #
+      # @param event [SolidusStripe::Webhook::Event]
+      def fail_payment(event)
+        payment = extract_payment_from_event(event)
+        return if payment.failed?
+
+        payment.failure!.tap do
+          SolidusStripe::LogEntries.payment_log(
+            payment,
+            success: true,
```

Comment:
Shouldn't this be `false`?

Reply:
Oh, you're right. Changed! Thanks for pointing out.

I also added another commit to test the success value for both handlers in the subscriber class.